main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
```

Comment:
Consider using a for loop starting here and have it loop 10 times. 🐶

Also for Python, we use # for comments. Unfortunately, this is not Javascript :(